papaya-web/src/main/java/pl/edu/agh/papaya/mappers/AvailabilityMapper.java

```diff
@@ -2,19 +2,36 @@
 
 import java.time.Duration;
 import java.util.Optional;
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Component;
 import pl.edu.agh.papaya.api.model.AvailabilityDto;
 import pl.edu.agh.papaya.model.Availability;
+import pl.edu.agh.papaya.service.sprint.SprintService;
 
 @Component
+@RequiredArgsConstructor
 public class AvailabilityMapper implements Mapper<Availability, AvailabilityDto> {
 
+    private final SprintService sprintService;
+
     @Override
     public AvailabilityDto mapToApi(Availability availability) {
+        double coefficient =
```

Comment:
Ok, lepiej, jak się liczy raz - ja bym i tak zrobiła to bez pola na `SprintService` z wariantem bez argumentu `coefficient`, który nie ustawia tych skalowanych wartości w ogóle (albo ustawia je tak samo jak nieskalowane) - te pola będą takie niekoniecznie często używane i tak - na przykład przy słaniu dostępności do wyświetlenia na froncie jak dla mnie nie ma sensu ich liczyć, ale jak uważasz

Reply:
jest jeszcze opcja zrobienia czegos w stylu `FatAvailabilityDto` w api dla rozróżnienia

Reply:
dobra, na razie zeby nie komplikowac to zrobie tak jak mowisz i to dodatkowe pole się doda gdzies tam glebiej

Reply:
kurde same problemy z tym są....

Reply:
i dlatego takie rzeczy się robi na froncie

nie jestem w stanie ładnie opóźnić ustawiania tego `effectiveTimeAvailable` bo potem to sie zagniezdza w userava... potem w list i by sie syf zrobil

tak samo nie da się ładnie zrobić mojego `mapToApiWithCoefficient` bo i tak to idzie przez UserAvailabilityMapper i tam by to nie miało sensu...

Prosze niech to zostanie jak jest i się pomysli w przyszłości jak zacznie to sprawiać problemy

Reply:
(A jednak nie liczy się raz - wydawało mi się, że masz zoverride'owane też `List<AvailabilityDto> mapToApi(List<Availability> availability, double coefficient)`, ale jednak nie, także w sumie ta prywatna metoda niewiele zmienia)

Ok, niech będzie, jak jest, później się pomyśli :+1: